examples/mongodbatlas_cloud_backup_snapshot_export_bucket/azure/main.tf

```diff
@@ -0,0 +1,30 @@
+resource "mongodbatlas_cloud_provider_access_setup" "setup_only" {
+  project_id    = var.project_id
+  provider_name = "AZURE"
+  azure_config {
+    atlas_azure_app_id   = var.azure_atlas_app_id
+    service_principal_id = azuread_service_principal.mongo.id
+    tenant_id            = var.tenant_id
+  }
+}
+
+resource "mongodbatlas_cloud_provider_access_authorization" "auth_role" {
+  project_id = var.project_id
+  role_id    = mongodbatlas_cloud_provider_access_setup.setup_only.role_id
+
+  azure {
+    atlas_azure_app_id   = var.azure_atlas_app_id
+    service_principal_id = azuread_service_principal.mongo.id
+    tenant_id            = var.tenant_id
+  }
+}
+
+
+resource "mongodbatlas_cloud_backup_snapshot_export_bucket" "test" {
```

Comment:
Can we add `README.md` files to these examples?

Reply:
Done